InfiniteRecharge/src/main/java/frc/util/ColourSensor.java

```diff
@@ -3,16 +3,38 @@
 import com.revrobotics.ColorMatch;
```

Comment:
would be nice to move to util folder but this can happen later

Reply:
Not an issue, but would probably be better off done in a VS code, as there would be a few errors in the imports/

Reply:
agreed